lib/services/compare.go

```diff
@@ -31,6 +31,7 @@ func CompareResources(resA, resB types.Resource) int {
 		ignoreProtoXXXFields(),
 		cmpopts.IgnoreFields(types.Metadata{}, "ID", "Revision"),
 		cmpopts.IgnoreFields(types.DatabaseV3{}, "Status"),
+		cmpopts.IgnoreFields(types.UserSpecV2{}, "Status", "CreatedBy"),
```

Comment:
Reviewers: I'm not sure of the impact this might have elsewhere. Any insights?

Reply:
I think this should be okay. I'm not overly worried about these, anyway.